pkg/arvo/app/time.hoon

```diff
@@ -1,23 +0,0 @@
-::
```

Comment:
This is a useful piece of standalone functionality, please restore.

```
> |start %time
activated app home/time
>=
> :time 1
>=
[%took ~s0..11ad]
```

Reply:
Added back.